client/src/style/options.scss

```diff
@@ -70,3 +70,7 @@
   margin-top: $spacing-s;
   transition: all 0.8s;
 }
+
```

Comment:
Can we change the class name to `.nowrap`? This may be reused for other stuff!

Reply:
The class name should now be updated to .nowrap.